org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/thermostat.js

```diff
@@ -0,0 +1,4 @@
+var AllJoynConverter = require("openT2T/converters/AllJoynConverter");
+
+module.exports = AllJoynConverter.readDeviceInterfacesFromFile(
+    require("path").join(__dirname, "thermostat.xml"))[0];
```

Comment:
>"thermostat.xml" [](start = 36, length = 16)

I think some people might prefer the name of the schema .xml to match the org.OpenT2T... name.